contracts/HundredVesting.sol

```diff
@@ -23,55 +23,50 @@ contract HundredVesting is Ownable {
         totalPeriod = totalPeriod / period;
     }
 
-    function getRemainedAmount (address vester) internal view returns (uint256) {
-        uint256 timestamp = addresses[vester].timestamp;
-        uint256 amount = (block.timestamp - timestamp) / timePerPeriod * addresses[vester].amount / totalPeriod;
-        return amount - addresses[vester].claimedAmount;
+    function getRemainedAmount (address vestedAddress) internal view returns (uint256) {
+        uint256 timestamp = addresses[vestedAddress].timestamp;
+        uint256 amount = (block.timestamp - timestamp) / timePerPeriod * addresses[vestedAddress].amount / totalPeriod - addresses[vestedAddress].claimedAmount;
+        return amount < 0 ? 0 : amount;
```

Comment:
`amount` will never be `< 0` as it is of type `uint256` so either switch to a signed type of check the values will not lead to a negative value before doing the calculation

Reply:
I was discussing it with vfat. I will update the code